proto/dex/tx.proto

```diff
@@ -36,7 +36,7 @@ message MsgDeposit {
   ]; 
   repeated int64 tickIndexes = 7;
   repeated uint64 feeIndexes = 8;
-  
+  bool autoswap = 9;
```

Comment:
can we put this within a DepositOptions field?

Reply:
Not sure what a DepositsOptions field is but assume it makes it an optional parameter (defaulted to false?). Think this is a good change if yeah.